proto/storage/cve.proto

```diff
@@ -161,6 +163,34 @@ message ImageCVE {
   CvssScoreVersion nvd_score_version = 12;
 }
 
+message ImageCVEV2 {
+  // keeping id similar construction for now during investigation.  It will include component and index
```

Comment:
One question about the CVE ID containing `componentID` + `index` . Would the index of a CVE in a component remain the same for repeated scans?

Reply:
No, it would not if the index changed for instance.  But that shouldn't matter too much as the way we load that data is delete/insert.  And the only piece of the ID that is currently used for any logic per se is the CVE itself.  Which I hope to make most of that go away.